pkg/test/matchers/proto.go

```diff
@@ -56,13 +57,13 @@ func (p *ProtoMatcher) yamls(actual interface{}) (string, string, error) {
 	actualProto := actual.(proto.Message)
 	actualYAML, err := util_proto.ToYAML(actualProto)
 	if err != nil {
-		return "", "", errors.New("Proto are not equal (could not parse actual proto to present differences)")
+		return "", "", fmt.Errorf("Proto are not equal (could not convert to YAML: %w)", err)
```

Comment:
I think ideally we want to use `github.com/pkg/errors` all over the project. I know there are still places with `errors` package, but I believe we should make it more consistent

Reply:
Consistency is always a winner, fixed!